wgpu/examples/bunnymark/main.rs

```diff
@@ -349,3 +349,19 @@ impl framework::Example for Example {
 fn main() {
     framework::run::<Example>("bunnymark");
 }
+
+#[test]
+fn bunnymark() {
+    framework::test::<Example>(
+        concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/examples/bunnymark/screenshot.png"
```

Comment:
would it make sense to pass only this part? i.e. can the `fn test()` implementation look up the `CARGO_MANIFEST_DIR` and do the path formation?

Reply:
Sure